docs/start-deployment.mdx

```diff
@@ -2,17 +2,28 @@
 title: Deployment
 ---
 
+export const DeploymentHelp = ({ step }) => {
```

Comment:
wondering if we should include the title too, in case the steps ever change, so we have a snapshot

Reply:
Yes, that's a good point. Would also make more sense for externals seeing the issues and greatly improve searchability of past issues. Will adapt!